packages/main/test/pages/RatingIndicator.html

```diff
@@ -68,6 +68,20 @@ <h3>RatingIndicator with title</h3>
 	<br>
 	<br>
 
+	<h3>RatingIndicator with required</h3>
+	<ui5-label id="label-required" for="rating-indicator-required">Required label</ui5-label>
+	<ui5-rating-indicator id="rating-indicator-required" required="true"></ui5-rating-indicator>
+	<br>
+	<br>
+	<br>
+
+	<h3>RatingIndicator with accessibleNameRef</h3>
+	<ui5-label id="label-acc-name-ref" for="rating-indicator-acc-name-ref">Some ACC label</ui5-label>
```

Comment:
can you add an example for accessibleName as well?

Reply:
done